pages/Page/index.jsx

```diff
@@ -62,14 +82,14 @@ class Page extends Component {
     }
 
     return (
-      <View
-        className={styles.container}
-        style={this.props.style}
-        title={pageConfig.title || ''}
-      >
-        <div className={styles.widgetWrapper}>
-          <Widgets components={widgets} widgets={pageConfig.widgets} />
-        </div>
+      <View className={styles.container} style={this.props.style} title={this.title}>
+        <Portal name={PAGE_CONTENT_BEFORE} />
+        <Portal name={PAGE_CONTENT} id={this.pageID}>
```

Comment:
`pageID` is wrong. It has to be `pageId`. And i think that you'd have to add in in the new `props` style to all portals of the page.